forgerock-auth/src/main/java/org/forgerock/android/auth/Config.java

```diff
@@ -107,13 +107,22 @@ public void setSharedPreferences(SharedPreferences sharedPreferences) {
         this.sharedPreferences = sharedPreferences;
     }
 
-    // We need to remove this in future, because the connected tests are depends on this. but the unit tests/integration tests are not depend on this.
-    @Deprecated
+    /**
+     * Load all the SDK configuration from the strings.xml
+     *
+     * @param appContext  The Application Context
+     */
```

Comment:
Actually i like to keep this method. this would be the easiest way to load the values from strings.xml. may be some customer may like to use it when they call after **reset()** method